app/src/main/java/com/github/fi3te/notificationcron/ui/NotificationCronAdapter.kt

```diff
@@ -68,6 +69,7 @@ class NotificationCronAdapter(private var data: List<NotificationCron>, private
             cronText.text = notificationCron.cron
             notificationTitleText.text = notificationCron.notificationTitle
             notificationTextText.text = notificationCron.notificationText
+            onClickURI.text = notificationCron.onClickURI
```

Comment:
Maybe this field should be hidden if it is not used (View.GONE)

Reply:
I tried it out initially but felt that having different heights for items in the list was a bit off.

On top of that the same isn't being done for the Notification text field.

Reply:
Very long notification texts are possible, which can lead to different heights. But you are right, using the fields differently would also be inconsistent.